editor/editor_dock_manager.cpp

```diff
@@ -184,8 +187,10 @@ void EditorDockManager::_update_docks_menu() {
 		docks_menu->set_item_icon(id, icon.is_valid() ? icon : default_icon);
 		if (!dock.value.open) {
 			docks_menu->set_item_icon_modulate(id, closed_icon_color_mod);
+			docks_menu->set_item_tooltip(id, TTR("Open dock."));
+		} else {
+			docks_menu->set_item_tooltip(id, TTR("Focus dock."));
```

Comment:
I think that, given it's a tooltip, these can afford to be a bit more articulated.
```suggestion
			docks_menu->set_item_tooltip(id, TTR("Open this dock."));
		} else {
			docks_menu->set_item_tooltip(id, TTR("Focus on this dock."));
```
Alternatively, instead of "this" it could say the name of the dock itself, but it doesn't matter that much.

Reply:
Using the name of the dock should fix it so the tooltips move when hovering over different items.

https://github.com/user-attachments/assets/3aefa81c-9d58-4989-84b1-3f941917208a

So I think I'll use the names instead of 'this'.

Reply:
Amusingly convenient for tooltip's questionable implementation